### PR TITLE
Update DSN ports and upgrade to april release

### PR DESCRIPTION
**Pull request by shamil-gadelshin**
This PR updates DSN ports from 50000->50001 and 30433->30434 for several scripts.

### CODE REVIEW

**Review by nazar-pc — COMMENTED** [review]
Well, since these are primitives, they will likely break devnet and whatever other networks we have by changing their ports as well, so I'm not 100% sure about this one

**Comment by shamil-gadelshin** [comment]
Since we're not sure about this PR - I'm converting it to the draft until we cancel it. 
Also, keep in mind if you're going to merge it after all that we need to add the "--protocol-version" CLI argument for bootstrap-node with the value equals to the genesis hash of gemini-3d

File: network-primitives/scripts/create_bootstrap_node_compose_file.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ services:
     restart: unless-stopped
     ports:
       - "30333:30333"
-      - "30433:30433"
```
Comment (vedhavyas) [inline network-primitives/scripts/create_bootstrap_node_compose_file.sh:39]: Are you sure about the this change? Looking at this PR - https://github.com/subspace/subspace/pull/1371/files
I think what you want is this instead
`30434:30433`

**Review by vedhavyas — COMMENTED** [review]
network-primitives should expose the dsn listen port with default as a variable through the config so that we can customise it just for gemini-3d. I can push those changes if you are not comfortable with terraform.

File: network-primitives/scripts/create_bootstrap_node_compose_file.sh (hunk 1 above)
Comment (shamil-gadelshin) [reply network-primitives/scripts/create_bootstrap_node_compose_file.sh:39]: Indeed. I wasn't going to change the default ports. Will fix it now.

**Comment by shamil-gadelshin** [comment]
I fixed the container ports back to the default (30433) and added `--protocol-version` (GENESIS_HASH) parameter to the bootstrap-application  - we need it after https://github.com/subspace/subspace/pull/1377 . However, I'm not aware of how do we set the environment variables.

**Comment by vedhavyas** [comment]
@shamil-gadelshin all good. I will take over from here to make necessary changes and get this in. Thanks!

File: network-primitives/scripts/create_bootstrap_node_compose_file.sh [hunk 2]
```diff
@@ -26,8 +26,13 @@ services:
     environment:
       - RUST_LOG=info
     ports:
-      - "50000:50000"
-    command: ["start", \$DSN_NODE_KEY, "/ip4/0.0.0.0/tcp/50000" ]
+      - "50001:50001"
```
Comment (nazar-pc) [inline network-primitives/scripts/create_bootstrap_node_compose_file.sh]: Shouldn't this be configurable as well?

File: network-primitives/variables.tf [hunk 3]
```diff
@@ -31,6 +31,7 @@ variable "full-node-config" {
     docker-tag          = string
     reserved-only       = bool
     prune               = bool
+    node-dsn-port       = string
```
Comment (nazar-pc) [inline network-primitives/variables.tf]: Is there a `number` type? Seems more appropriate for the port.

**Review by nazar-pc — DISMISSED** [review]

File: network-primitives/scripts/create_bootstrap_node_compose_file.sh (hunk 2 above)
Comment (vedhavyas) [reply network-primitives/scripts/create_bootstrap_node_compose_file.sh]: actually, yes. Good catch

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**